src/frontend/src/flows/register/finish.ts

```diff
@@ -119,7 +116,6 @@ export const displayUserNumberWarmup = (): OmitParams<
   typeof displayUserNumber,
   "identityBackground"
 > => {
-  const identityBackground = loadIdentityBackground();
```

Comment:
This would have triggered some preloading of the image in the client, right?
So presumably, the image is now loaded later. Is this noticeable for the clients?

Reply:
Good point. It might be noticeable. I didn't realize this functionality. Thanks for bringing it up.

Let me close this PR and I'll fix the issue in another way.